kintone-ios-sdk/Classes/module/record/Record.swift

```diff
@@ -12,6 +12,9 @@ open class Record: NSObject {
     private var connection: Connection?
     private let parser = RecordParser()
     
+    private let LIMIT_REQUEST_PER_BULK = 20;
+    private let LIMIT_RECORD_PER_REQUEST = 100;
```

Comment:
Please move two constants to https://github.com/kintone/kintone-ios-sdk/blob/SSR-479-getAllRecordsByQuery/kintone-ios-sdk/Classes/module/record/RecordConstants.swift
You can merge branch SSR-479-getAllRecordsByQuery.
Thank you.